src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -186,16 +214,19 @@ When `CT=0`, the capability authorizes access to a region of memory as defined b
 
 [#sentry_cap,reftext="sentry capability"]
 Sentry capabilities::
-Capabilities with `CT=1` describe function entry points, known as sealed entry (sentry) capabilities.
-They cannot be dereferenced to access memory and are only allow very limited changes to the fields in the capability, which is restricted to reducing the <<section_cap_level, CL>> field using <<ACPERM>>.
-Modifying any other fields with <<ACPERM>>, or any other instruction, clears the valid tag of the output capability.
-However, control-flow instruction can be used to jump to a sentry capability to begin executing the instructions it references.
+Capabilities with `CT=1` are sealed against modification.
+They cannot be dereferenced to access memory and it is not permitted to change the address, bounds or permissions.
+Modifications are restricted to _reducing_ the <<section_cap_level, CL>> field using <<ACPERM>>.
++
+Sealed entries (also known as sentries) describe secure function entry points.
+They are used as the target of <<JALR_CHERI>>, which must have a zero offset for the use of a sentry to be valid.
++
+Sentry capabilities can establish a form of control-flow integrity between mutually distrusting code.
+<<JALR_CHERI>> with zero offset automatically unseals a sentry target capability and installs it in the <<pcc,program counter capability>>.
++
+<<JALR_CHERI>> also seals the return register, so the callee can return to the caller but cannot access the memory pointed at by the return register, and cannot return elsewhere.
 +
-NOTE: Sentry capabilities can establish a form of control-flow integrity between mutually distrusting code.
-For example, the <<JALR_CHERI>>
-instruction with zero offset automatically unseals a sentry target capability and installs it in the
-<<pcc,program counter capability>>.
-The jump-and-link instructions also seal the return register, so the callee can return to the caller but cannot access the memory pointed at by the return register.
+#Tariq: I'm concerned by this note - as it doesn't seem to fit the definition of CBLD#
```

Comment:
See my comment on the issue, we just need a SCHI first.

Reply:
ok thanks